### ISSUE TITLE
Updated logic for results metrics

### ISSUE THREAD

**Issue by rx294** [mitre/heimdall#85]
1. any kind of error -> Profile Error
2. impact 0 -> N/A (except if #1)
3. has any failed tests -> Fail whole control (except if #1 or #2)
4. has skip and pass -> Pass whole control (except if #1 or #2)
5. all tests within a control pass -> Pass (except if #1 or #2)
6. only skips -> Requires Manual Review (except if #1 or #2)

@ejaronne @aaronlippold

**Comment by ejaronne** [comment]
Yes, I believe it should.  It means something went wrong with could and/or access, so it should be an error.

**Comment by ejaronne** [comment]
Please check the ubuntu circle on http://xk3r.hatchboxapp.com/circles/5 
These are 3 different overlays against the ubuntu baseline. when loaded into original Heimdall-lite, they manifest different scores, but in heimdall server they all show the same scores. is this an anamoly related to the recent https://github.com/mitre/heimdall/issues/85 closure?
